integrity-backend/input-browsertrix-examples/3e11cc57daf3bad8375935cad4878123acc8d769551ff90f1b1bb0dc597-meta-content.json

```diff
@@ -0,0 +1,21 @@
+{
+  "contentMetadata":
+  {
+    "name": "Web archive of @rt_com Twitter",
+    "mime": "application/wacz",
```

Comment:
@ikreymer does this look right?

Reply:
yes, mime type should be good. Will think about if there is anything else that should be included, such as the main page url and timestamp (assuming these are always one-page wacz files)

Reply:
All that ins `extras` as they are specific to type of asset. Those get registered to blockchain. Things in `private` stay encrypted and not registered publicly on chain.